services/workflows-service/src/business-report/business-report.service.ts

```diff
@@ -156,6 +156,17 @@ export class BusinessReportService {
       filePath: merchantSheet.path,
       schema: BusinessReportRequestSchema,
       logger: this.logger,
+      cast: (value, context) => {
+        const headerToSchemaKeyMap: Record<string, string> = {
+          'Website URL': 'websiteUrl',
+          'Company Name/Registered Merchant Name (Optional)': 'merchantName',
+          'Merchant ID (Optional)': 'correlationId',
+        };
+
+        if (!context.header || !headerToSchemaKeyMap[value]) return value;
```

Comment:
Please make sure that Lint and Prettier run on your IDE during "On Save"
```suggestion
        if (!context.header || !headerToSchemaKeyMap[value]) {
          return value;
        }
```

Reply:
Done.